libs/docs/core/calendar/examples/calendar-multi-example.component.ts

```diff
@@ -0,0 +1,64 @@
+import { DatePipe, JsonPipe } from '@angular/common';
+import { Component } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+import { ButtonComponent } from '@fundamental-ngx/core/button';
+import { CalendarComponent } from '@fundamental-ngx/core/calendar';
+import {
+    DATE_TIME_FORMATS,
+    DatetimeAdapter,
+    FD_DATETIME_FORMATS,
+    FdDate,
+    FdDatetimeAdapter,
+    FdDatetimeModule
+} from '@fundamental-ngx/core/datetime';
+
+@Component({
+    selector: 'fd-calendar-multi-example',
+    template: `
+        <fd-calendar calType="multi" [(ngModel)]="dates" [showWeekNumbers]="true"></fd-calendar>
+        <br />
+
+        <div>
+            Selected Dates:<br />
+            @for (date of dates; track date.day) {
```

Comment:
In your case, if you are using date.day as the trackBy function, Angular will track items based on the day number only. This means that if the same day number (e.g., the 1st) appears in both the current month and the next month, Angular might treat them as the same item. This can cause issues such as incorrect rendering, lost updates, or unexpected behavior.

To avoid this, you should use a more unique identifier for the trackBy function. A combination of the year, month, and day would ensure each date is uniquely tracked.